src/Microsoft.AspNetCore.Authentication/Events/EventResultState.cs

```diff
@@ -11,13 +11,18 @@ public enum EventResultState
         Continue,
 
         /// <summary>
+        /// Bypass the default logic.
+        /// </summary>
+        BypassDefaultLogic,
```

Comment:
I think maybe Noop is better than bypass.  That's what you want right?  For the event to be an explicit Noop